api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -25,17 +25,19 @@
 import org.pmiops.workbench.cdr.CdrVersionService;
 import org.pmiops.workbench.cdr.dao.CBCriteriaAttributeDao;
 import org.pmiops.workbench.cdr.dao.CBCriteriaDao;
-import org.pmiops.workbench.cdr.model.CBCriteria;
-import org.pmiops.workbench.cdr.model.CBCriteriaAttribute;
-import org.pmiops.workbench.cdr.model.MenuOption;
+import org.pmiops.workbench.cdr.model.DbCriteria;
```

Comment:
Oh, so these are in `cdr.model` and not `db.model`. That explains why I missed them.Do you expect other DB models to be stored with their most closely related packages?

Reply:
Yes. Some background on the split between db.model and cdr.model. Anything that lives in db.model is in the cloud mysql workbench db schema(1 schema for all cdrs) and anything in cdr.model is in the cloud mysql cdr(multiple cdr versions) schemas. We keep this separation  because when querying any model with the cdr.model package, we have to switch cdr db schemas on the fly per client request.